VogtPayroll4/Employee.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll4
+{
+    class Employee
+    {
+        public string empName;
+        public string empID;
+        public string hourlyRate;
+        public string hoursWorked;
+
+        public Employee(string empName, string empID, string hourlyRate, string hoursWorked)
```

Comment:
Pass in the empId, hourlyRate, hoursWorked as int. You should have the parsing code moved into something like a EmployeePayrollConsoleReader